features/admin.console-settings.v1/components/console-roles/console-roles-edit/console-role-permissions.tsx

```diff
@@ -123,6 +128,11 @@ const ConsoleRolePermissions: FunctionComponent<ConsoleRolePermissionsProps> = (
             cloneDeep(tenantAPIResourceCollections);
         const filteringAPIResourceCollectionNames: string[] = [];
 
+        if(!useSCIM2RoleAPIV3) {
```

Comment:
Shouldn't this be as follows?

```suggestion
        if(useSCIM2RoleAPIV3) {
```